src/sprite_battler.cpp

```diff
@@ -259,7 +259,7 @@ void Sprite_Battler::CreateSprite() {
 
 	SetX(battler->GetBattleX());
 	SetY(battler->GetBattleY());
-	SetZ(battler->GetBattleY()); // Not a typo
+	SetZ(PriorityBattle + battler->GetBattleY()); // Not a typo
```

Comment:
The typo was about using GetBattleY instead of GetBattleZ (which doesn't exist). Yes these lines basicly mean that battler with a lower Y (so higher in the scene) appear behind battlers with higher Y.